src/overlays/actors/ovl_En_Rd/z_en_rd.c

```diff
@@ -1,3 +1,9 @@
+/*
+ * File: z_en_rd.c
+ * Overlay: ovl_En_Rd
+ * Description: Redead / Gibdo
```

Comment:
Is this really both redead and gibdo? I'd have thought they'd be different actors?

Reply:
I just copy what's on the spreadsheet.